api/v1/handlers.go

```diff
@@ -3902,7 +3654,7 @@ func (s *Server) handleDeleteContentFromCollection(c echo.Context, u *util.User)
 	return c.NoContent(http.StatusOK)
 }
 
-func (s *Server) tracingMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
+func (s *apiV1) tracingMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
```

Comment:
you can remove this since you have it in util/tracing_middleware.go and configure it in api_engine already